opm/grid/cpgrid/Intersection.hpp

```diff
@@ -191,12 +191,28 @@ namespace Dune
                 return geometry().type();
             }
 
-            /// Local index of codim 1 entity in the inside() entity
+            /// \brief Local index of codim 1 entity in the inside() entity
             /// where intersection is contained in.
+            ///
+            /// Note that CpGrid does not support codim-1 entities and return
+            /// the index of the face of the underlying cartesian grid assuming that
+            /// all faces are perpendicular to the axis:
+            /// For faces perpendicular to x-axis: 0 (normal points contrary to axis) 1 (else)
+            /// For faces perpendicular to x-axis: 2 (normal points contrary to axis) 3 (else)
```

Comment:
This should be y-axis (and z-axis below), similar for the next block.

Also: the return values are determined by topological orientation, not by the direction the normal points in.